### PR TITLE
Use GetView in addition to FindView for view discovery

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes https://github.com/aspnet/Entropy/issues/260

### CODE REVIEW

File: samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs [hunk 1]
```diff
@@ -34,15 +36,7 @@ public class RazorViewToStringRenderer
         public async Task<string> RenderViewToStringAsync<TModel>(string name, TModel model)
```
Comment (ajaybhargavb) [inline samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs]: Could you rename the param to `viewPath`? Having `name` doesn't make sense.

Comment (pranavkm) [inline samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs]: On second thought, the current name matches what we have in `ViewResult`: https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewResult.cs#L30. I'll change it to `viewName` and call it a day (particularly given that it's not necessarily a path all the time).

Comment (ajaybhargavb) [inline samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs]: Sure. My concern was mostly about having to specify the `.cshtml` part of the path.

Comment (pranavkm) [inline samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs]: We don't always require the extension `RenderViewToStringAsync("View", null)` would work just as fine. However, there are oddities with the paths we generate that prevent it from working with precompiled views. If / when we choose to fix it, we should be able to use either interchangeably.

**Pull request closed without merging**